src/datadog_api_client/v1/model/aws_account.py

```diff
@@ -50,26 +50,26 @@ def __init__(self, *args, **kwargs):
         :param account_id: Your AWS Account ID without dashes.
         :type account_id: str, optional
 
-        :param account_specific_namespace_rules: An object, (in the form `{"namespace1":true/false, "namespace2":true/false}`),
+        :param account_specific_namespace_rules: An object, (in the form ``{"namespace1":true/false, "namespace2":true/false}``\ ),
             that enables or disables metric collection for specific AWS namespaces for this
             AWS account only.
         :type account_specific_namespace_rules: {str: (bool,)}, optional
 
-        :param cspm_resource_collection_enabled: Whether Datadog collects cloud security posture management resources from your AWS account. This includes additional resources not covered under the general `resource_collection`.
+        :param cspm_resource_collection_enabled: Whether Datadog collects cloud security posture management resources from your AWS account. This includes additional resources not covered under the general ``resource_collection``.
         :type cspm_resource_collection_enabled: bool, optional
 
         :param excluded_regions: An array of AWS regions to exclude from metrics collection.
         :type excluded_regions: [str], optional
 
-        :param filter_tags: The array of EC2 tags (in the form `key:value`) defines a filter that Datadog uses when collecting metrics from EC2.
-            Wildcards, such as `?` (for single characters) and `*` (for multiple characters) can also be used.
+        :param filter_tags: The array of EC2 tags (in the form ``key:value``\ ) defines a filter that Datadog uses when collecting metrics from EC2.
```

Comment:
```suggestion
        :param filter_tags: The array of EC2 tags (in the form ``key:value`` ) defines a filter that Datadog uses when collecting metrics from EC2.
```
Looks like there is an additional `\`